matlab/SummaryPlotter.m.orig

```diff
@@ -0,0 +1,79 @@
+%% SummaryPlotter
```

Comment:
This looks like the result of a merge conflict. Is SummaryPlotter.m OK after the merge? If it is, this file should be deleted and removed from the repository.